### ISSUE TITLE
Arabic ligature U+FDFD needs some characters repositioning/adjustment in NotoNaskhArabic and NotoNaskhArabicUI

### ISSUE THREAD

**Issue by nizarsq** [googlefonts/noto-fonts#1874]
Arabic ligature U+FDFD ARABIC LIGATURE BISMILLAH AR-RAHMAN AR-RAHEEM needs some adjustment to where characters set or some adjustment to character U+0645 (م) length in both NotoNaskhArabic and NotoNaskhArabicUI. Currently character U+0645 (م) bottom attached to the top part of character U+0646 (ن) this makes them look like the character U+0644 (ل) with dot.

<b>Issue found in: </b> 
NotoNaskhArabic Version 2.000
https://github.com/googlefonts/noto-fonts/tree/master/unhinted/ttf/NotoNaskhArabic/NotoNaskhArabic-Regular.ttf
NotoNaskhArabicUI Version 2.000
https://github.com/googlefonts/noto-fonts/tree/master/unhinted/ttf/NotoNaskhArabicUI/NotoNaskhArabicUI-Regular.ttf

<b>Characters data:</b>
﷽

<b>Screenshot:</b>
<img width="1131" alt="Screen Shot 2020-10-01 at 11 04 34 AM" src="https://user-images.githubusercontent.com/22158882/94847631-b2f22e00-03d7-11eb-8793-52c97667842d.png">

Other Noto fonts rendering (Different calligraphy):
<img width="856" alt="Screen Shot 2020-10-01 at 11 18 53 AM" src="https://user-images.githubusercontent.com/22158882/94847789-e59c2680-03d7-11eb-8534-ab1801007f07.png">

**Comment by nizarsq** [comment]
I believe moving (AlLAH) and (ALRHMAN) to the right to make them centered with (BISM) and (ALRAHEM) will solve this issue and will solve another issue with unbalanced appearance of U+FDFD ligature.
<img width="377" alt="Screen Shot 2020-10-01 at 1 32 20 PM" src="https://user-images.githubusercontent.com/22158882/94860439-eab6a100-03ea-11eb-9db9-07f1242ed35f.png">

**Comment by khaledhosny** [comment]
I like to add that this ligature is pretty bad in general. The text should be one line, since this ligature is almost always used at the benign of text on a line of its own, and the Noto version is not suitable for this use at all. Compare it to the version in many other Arabic fonts.
<img width="1342" alt="image" src="https://user-images.githubusercontent.com/93914/102027305-dd833e80-3dab-11eb-855d-61a7b53a96e6.png">

**Comment by nizarsq** [comment]
@khaledhosny I agree, but  in NotoNaskhArabic  @marekjez86 prefer to keep FDFD compacted since it goes with Android. I tried to make it to look little better, but I'm still not satisfied how it came out (properly will revise the changes). I don't have much room to make any other changes. Here how the initial changes look like. Any suggestions are welcome :)
<img width="905" alt="Screen Shot 2020-12-13 at 3 50 52 PM" src="https://user-images.githubusercontent.com/22158882/102027725-fd3e4680-3d5a-11eb-9ced-b7564a474f6d.png">


I changed the design in NotoSansArabic fonts from the compacted design, currently look like this
<img width="405" alt="Screen Shot 2020-12-13 at 3 42 04 PM" src="https://user-images.githubusercontent.com/22158882/102027547-c287de80-3d59-11eb-9720-6cddae46700d.png">
